dotenv.example

```diff
@@ -63,3 +63,7 @@ OSEM_SMTP_OPENSSL_VERIFY_MODE=""
 
 # Enable the usage of the devise ichain plugin
 OSEM_ICHAIN_ENABLED=false
+
+# ReCAPTCHA keys
+RECAPTCHA_SITE_KEY=""
```

Comment:
please prefix them with `OSEM_`

Reply:
The key names are defined [upstream](https://github.com/ambethia/recaptcha#rails-installation) in the rubygem; while it _can_ be configured more explictly, that means adding yet another initializer, etc., I'd rather stick with the default upstream behavior.